apiserver/apiserver.go

```diff
@@ -428,17 +409,18 @@ func newServer(ctx context.Context, cfg ServerConfig) (_ *Server, err error) {
 		controllerConfigService, macaroonService,
 	)
 	if err != nil {
-		unsubscribeControllerConfig()
 		return nil, fmt.Errorf("creating offer auth context: %w", err)
 	}
 
 	ready := make(chan struct{})
-	srv.tomb.Go(func() error {
-		defer srv.logSink.Close()
```

Comment:
We should never close the log sink here, that is a mistake. The logsink has it's own worker now, so bouncing the API server would have caused chaos.